src/support.sch

```diff
@@ -136,6 +143,26 @@
         </rule>
     </pattern>
 
+    <pattern id="fig">
+        <rule context="fig">
+            <assert test="graphic[(@mimetype='image' and @mime-subtype='jpeg') or ('.jpg'=substring(@xlink:href, string-length(@xlink:href)-3))]" role="warn">
+                &lt;<name/>&gt; without a JPEG &lt;graphic&gt; is ignored.
+            </assert>
+        </rule>
+    </pattern>
+
+    <pattern id="fig_parent">
+        <rule context="fig[parent::*]">
+            <let name="parent" value="name(..)"/>
+            <assert test="
+                $parent='body'
+                or $parent='sec'
+            " role="warn">
+                &lt;<name/>&gt; in &lt;<value-of select="$parent"/>&gt; is ignored.
+            </assert>
+        </rule>
```

Comment:
Presuming it's limited to `body` and `sec`, since that's all Libero can currently handle, but we (eLife) currently also allow `fig` in`fig-group`, `boxed-text` and `p` (although the last one I could see a case for mandating against).

Reply:
We'll definitely be adding support for `<fig-group>` and `<boxed-text>` (and probably others); `p` is unclear as HTML doesn't work in that way. Might be that https://github.com/libero/libero/issues/131 has a step to rewrite it.